### ISSUE TITLE
[Accessibility]: Confusing associated code intelligence record components

### ISSUE THREAD

**Issue by efritz** [sourcegraph/sourcegraph#35268]
### Audit type

Screen reader navigation

### User journey audit issue

#33513

### Problem description

![image](https://user-images.githubusercontent.com/103087/167728342-c5945678-daa3-4100-a690-f84cb93c702d.png)

In the code intelligence auto indexing job detail view, an associated upload is shown. There is too much context here about timings which don't really matter. This was very distracting when navigating.

Similarly, the phrasing "This job uploaded an index" is actually referring to what we call an _upload_, which ... is just very unfortunate naming.

### Expected behavior

This entire line could be simplified to a single sentence, or at least with fewer non-visual decorations.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by Strum355** [comment]
https://github.com/sourcegraph/sourcegraph/pull/35298 addresses the point on the last phrasing

**Comment by macraig** [comment]
Closing as identified issues have been completed
